app-script-payslip-sender/PayslipSender.js

```diff
@@ -0,0 +1,26 @@
+function PayslipSender() {
```

Comment:
better to follow the standard naming conventions for functions

should be payslipSender() in camel case.

https://www.30secondsofcode.org/js/s/naming-conventions/